unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/TextShape/ECSTextShapeComponentHandler.cs

```diff
@@ -32,25 +33,36 @@ public ECSTextShapeComponentHandler(AssetPromiseKeeper_Font fontPromiseKeeper, I
         this.renderersInternalComponent = renderersInternalComponent;
     }
 
+    private void OnTextRendererUpdated(TMP_TextInfo tmproInfo)
+    {
+        renderersInternalComponent.RemoveFor(scene, entity);
+        renderersInternalComponent.AddRenderer(scene, entity, textRenderer);
```

Comment:
why are removing the component for that entity and on the next line setting the renderers for the component?